platform/admin/ui/src/main/webapp/js/models/installer/CertsModel.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, location*/
+define([
+    'marionette',
+    'underscore',
+    'backbone',
+    'jquery'
+], function (Marionette, _, Backbone, $) {
+
+    var CertsModel = Backbone.Model.extend({
+        defaults: {
+            hostName: undefined,
```

Comment:
We have not done this in the past but I like this.  Makes is easy to see what fields we expect to be in the model.